src/MixedQuoter.sol

```diff
@@ -186,12 +199,35 @@ contract MixedQuoter is IMixedQuoter, IPancakeV3SwapCallback {
     /**
      * Mixed *************************************************
      */
+    /// @dev All swap results will influence the outcome of subsequent swaps within the same pool
+    function quoteMixedExactInputNotIsolation(
```

Comment:
(optional as this opinion varies)

have some other potential names for this function

```
- quoteMixedExactInputNonIsolated
- quoteMixedExactInputLinked
- quoteMixedExactInputSharedContext
```

can see if you feel they might be better, ok to stick with `quoteMixedExactInputNotIsolation` as well

Reply:
thanks, will use quoteMixedExactInputSharedContext